userspace/libsinsp/user.cpp

```diff
@@ -41,11 +41,13 @@ limitations under the License.
 #define HAVE_FGET__ENT
 #endif
 
+static libsinsp::procfs_utils::ns_helper *m_ns_helper;
+
```

Comment:
```suggestion
static libsinsp::procfs_utils::ns_helper *s_ns_helper;

```

Let's not name it `m_*`.